changelog/unreleased/3484-1

```diff
@@ -0,0 +1,7 @@
+Change: Make settings available in phoenix
```

Comment:
"Enhancement" might be better in this case?

Reply:
Nope, it's a change. ;-) Introduces settings to phoenix => entirely new feature.

Reply:
Exactly, a new feature which doesn't change any existing functionality